index.html

```diff
@@ -2627,6 +2704,102 @@ <h2>Protocol Binding</h2>
         MUST implement this protocol binding.
         </span>
       </p>
+      <p>
+        The examples provided throughout this section describe how a Consumer
+        would communicate with a Web Thing which produces the following Thing
+        Description:
+      </p>
+      <pre class="example">
+        {
+          "@context": "https://www.w3.org/2019/wot/td/v1",
+          "id": "https://mywebthingserver.com/things/lamp",
+          "profile": [
+            "https://www.w3.org/2022/wot/profile/http-baseline/v1",
+            "https://www.w3.org/2022/wot/profile/http-sse/v1",
+          ],
+          "base": "https://mywebthingserver.com/things/lamp/",
+          "title": "My Lamp",
+          "description": "A web connected lamp",
+          "securityDefinitions": {
+            "oauth2": {
+              "scheme": "oauth2",
+              "flow": "code",
+              "authorization": "https://mywebthingserver.com/oauth/authorize",
+              "token": "https://mywebthingserver.com/oauth/token"
+            }
+          },
+          "security": "oauth2",
+          "properties": {
+            "on": {
+              "type": "boolean",
+              "title": "On/Off",
+              "description": "Whether the lamp is turned on",
+              "forms": [{"href": "./properties/on"}]
+            },
+            "level" : {
+              "type": "integer",
+              "title": "Brightness",
+              "description": "The level of light from 0-100",
+              "unit": "percent",
+              "minimum" : 0,
+              "maximum" : 100,
+              "forms": [{"href": "./properties/level"}]
+            }
+          },
+          "actions": {
+            "fade": {
+              "title": "Fade",
+              "description": "Fade the lamp to a given level",
+              "input": {
+                "type": "object",
+                "properties": {
+                  "level": {
+                    "type": "integer",
+                    "minimum": 0,
+                    "maximum": 100,
+                    "unit": "percent"
+                  },
+                  "duration": {
+                    "type": "integer",
+                    "minimum": 0,
+                    "unit": "milliseconds"
+                  }
+                }
+              },
+              "forms": [{"href": "./actions/fade"}]
+            }
+          },
+          "events": {
+            "overheated": {
+              "title": "Overheated",
+              "data": {
+                "type": "number",
+                "unit": "degree celsius"
```

Comment:
please consider using a well known unit identifier, e.g. from QUDT.
Need to use a QUDT context file.
How to get a renderable unit? QUDT lookup or embed in the TD?

Reply:
I think this requires further discussion in #29 as to whether the profile mandates a particular unit ontology.

In WebThings we use the long form of [SI units](https://www.bipm.org/documents/20126/41483022/SI-Brochure-9-EN.pdf/2d2b50bf-f2b4-9661-f402-5f9d66e4b507?version=1.10&t=1632138671693&download=true) for the values of `unit` members, e.g. "volt", "ohm", "lux", "lumen", "hertz", "watt" etc.

Reply:
Arch call on May 11th:
Agree to merge this PR, discussion should continue in #29 - We could use SI units and potentially also permit QUDT with a prefix. Everything could be converted to SI units.